packages/react-scripts/scripts/build.js

```diff
@@ -112,7 +112,7 @@ function build(previousFileSizes) {
       if (messages.errors.length) {
         return reject(new Error(messages.errors.join('\n\n')));
       }
-      if (process.env.CI && messages.warnings.length) {
+      if (process.env.CI && process.env.CI.toLowerCase() === 'true' && messages.warnings.length) {
```

Comment:
I'm not sure about real practice of using this variable, I think we need some discussion first.